pyproject.toml

```diff
@@ -30,10 +30,13 @@ requires-python = ">=3.10"
 # Run 'tox -e deps' after making changes here. This will update requirement files.
 # Make sure to list one dependency per line.
 dependencies = [
-  "sciline>=24.02",
-  "scipp>=24.01",
-  "scippneutron>=24.01",
-  "scippnexus>=23.12",
+  "sciline==24.6.3",
+  "scipp==24.9.1",
+  "scippneutron==24.8.0",
+  "scippnexus==24.9.0",
+  "choppera==0.1.4",
+  "loguru==0.7.2",
+  "tqdm==4.66.5",
```

Comment:
We shouldn't lock the versions here, this is used to create the wheels and they should be usually just be lower pinned. `tox -e deps` will create the locked requirements in `requirements/*.txt` files.